apps/omg_utils/lib/omg_utils/http_rpc/response.ex

```diff
@@ -142,4 +138,13 @@ defmodule OMG.Utils.HttpRPC.Response do
         "watcher"
     end
   end
+  
+  defp get_vsn(service) when is_atom(service) do
+    with {:ok, vsn} <- :application.get_key(service, :vsn) do
+      vsn
+    else
+      _ ->
+        '0.0.0'
```

Comment:
this is not the right approach.

Reply:
Problem is that `:code.is_loaded(OMG.ChildChainRPC)` isn't right approach in all contexts we're running rpc servers, which includes:
* simple perf test where not full child_chain app is provisioned, but rather the pieces test need,
* ByzantineEventsTest - fixture `omg_watcher` which starts Watcher via Exexec

What's here just lowers the impact of bug introduced in `add_version`. Let's find a better solution in separate issue.